APPLICATION_PROGRESS.md

```diff
@@ -422,16 +422,17 @@ the sdk
 <details>
 <summary>Over the counter (OTC) Swap Predicate</summary>
 
-<h3>Contracts</h3>
+<h3>Contracts✅</h3>
 
-<h3>User Interface</h3>
+<h3>User Interface❌</h3>
```

Comment:
```suggestion
<h3>User Interface</h3>
```
If something has yet to be added then I wouldn't label it with a cross. I would only do that if the feature exists but there are significant enough issues that it should be marked as "not there yet" as opposed to "good enough for now / complete"

Reply:
Yeah I just pushed a new commit. Added ticks where feature is unimplemented but there is no intention for it to be.